GAE/src/org/waterforpeople/mapping/app/web/TestHarnessServlet.java

```diff
@@ -373,12 +374,12 @@ public void doGet(HttpServletRequest req, HttpServletResponse resp) {
         }
     }
 
-    private void setupTestUser() {
+    private void setupTestUser(String email) {
```

Comment:
I think we can remove this entire method. It was used initially when setting up test accounts for the local server but now we have sample data that includes test users so this is no longer necessary.

Reply:
For testing the Google integration, I am creating a new user that matches my Gmail account.

None of the tests accounts are admin accounts, so none can create new users.